src/kili/queries/lock/__init__.py

```diff
@@ -29,6 +28,10 @@ def __init__(self, auth):
 
     # pylint: disable=dangerous-default-value
     @typechecked
+    @deprecate(
+        "locks method is for internal use only and will be removed from 01/02/2013",
```

Comment:
I don't think there would be a use case for locks. We can skip the deprecation period.

Reply:
We want to keep it internally but I deprecated it for the sake of not introducing a breaking change without a deprecation period. 
I checked in Apollo, it has not been called in the last 3 months.
If you feel safe not doing a deprecation period, I follow you.